match.go

```diff
@@ -67,7 +68,7 @@ type globMatcher struct {
 }
 
 func (m *globMatcher) Match(name string) bool {
-	matches, err := filepath.Match(m.glob, name)
+	matches, err := doublestar.Match(m.glob, name)
```

Comment:
Shouldn't this be doublestar.PathMatch?

Reply:
Sorry, completely missed your review. Fixed.

Reply:
@cespare could you review? :-)